pymc/model_graph.py

```diff
@@ -163,7 +178,21 @@ def _make_node(self, var_name, graph, *, nx=False, cluster=False, formatting: st
                 shape = "ellipse"
                 style = None
             symbol = v.owner.op.__class__.__name__
-            if symbol.endswith("RV"):
+            if symbol == "MarginalMixtureRV":
+                components = v.owner.inputs[2:]
+                if check_zip_graph_from_components(components):
+                    # [12:-2] because the component name are of the form "Unmeasurable...RV"
+                    component_names = [var.owner.op.__class__.__name__[12:-2] for var in components]
+                    component_names.remove("DiracDelta")
+                    symbol = f"ZeroInflated{component_names[0]}"
+                elif len(components) == 1:
+                    symbol = f"{components[0].owner.op.__class__.__name__[12:-2]}Mixture"
+                else:
+                    symbol = symbol[:-2]  # just MarginalMixture
```

Comment:
If a mixture has two components that could still be nice to show as `X-YMixture`. What do you think?

Reply:
I think that this would be a good idea!